### PR TITLE
[#222] Return question personal-data

### PR DESCRIPTION
**Pull request by tangrammer**
Relates #222 
Besides returning question.personal-data this PR updates akvo-flow version in project.clj (updating a api change in com.gallatinsystems.survey.dao.SurveyDAO https://github.com/akvo/akvo-flow-api/pull/223/files#diff-03a03643d1e5bf683c06915a4313c418R70)

### CODE REVIEW

File: api/src/clojure/org/akvo/flow_api/datastore/survey.clj [hunk 1]
```diff
@@ -46,6 +46,7 @@
       :type type*
       :order (.getOrder question)
       :variable-name (.getVariableName question)
+      :personal-data (.getPersonalData question)
```
Comment (muloem) [inline api/src/clojure/org/akvo/flow_api/datastore/survey.clj:49]: In the UI the field is set as `holdPersonalData`? Maybe confirm with Marvin about this if you want it to match or you will have to translate from one field to the other

Comment (tangrammer) [reply api/src/clojure/org/akvo/flow_api/datastore/survey.clj:49]: we already changed it to `personalData`

**Review by muloem — COMMENTED** [review]
ready once broken build is sorted out. 👍

**Comment by tangrammer** [comment]
@muloem fixed build 😅

**Review by muloem — APPROVED** [review]

**Pull request merged**